client/src/pages/dashboard/Dashboard.js

```diff
@@ -27,8 +28,12 @@ class Dashboard extends Component {
 
   componentDidMount() {
     getUser().then(response => {
+      console.log(response)
       if (response.data.user) {
-        this.setState({ username: response.data.user.username }, () => {
+        this.setState({
+          username: response.data.user.username,
+          userId: response.data.user._id
```

Comment:
Upon page load, we can grab the userId from this function and store it as state for use later.